e2e-tests/gef/cypress/integration/unissued-facilities-AIN.spec.js

```diff
@@ -0,0 +1,523 @@
+      aboutFacilityUnissued.errorSummary().contains('Enter the date you issued the facility to the exporter');
+      aboutFacilityUnissued.errorSummary().contains('Select if you want UKEF cover to start on the day you issue the facility');
+      aboutFacilityUnissued.errorSummary().contains('Enter a cover end date');
+      aboutFacilityUnissued.issueDateDay().type(fourDaysAgoDay);
+      aboutFacilityUnissued.issueDateMonth().type(fourDaysAgoMonth);
+      aboutFacilityUnissued.issueDateYear().type(fourDaysAgoYear);
+      aboutFacilityUnissued.continueButton().click();
+      aboutFacilityUnissued.issueDateError().contains('The issue date must not be before the date of the inclusion notice submission date');
+      aboutFacilityUnissued.errorSummary().contains('The issue date must not be before the date of the inclusion notice submission date');
+      aboutFacilityUnissued.issueDateDay().clear();
+      aboutFacilityUnissued.issueDateMonth().clear();
+      aboutFacilityUnissued.issueDateYear().clear();
+      aboutFacilityUnissued.issueDateDay().type(threeMonthsOneDayDay);
+      aboutFacilityUnissued.issueDateMonth().type(threeMonthsOneDayMonth);
+      aboutFacilityUnissued.issueDateYear().type(threeMonthsOneDayYear);
+      aboutFacilityUnissued.continueButton().click();
+      aboutFacilityUnissued.issueDateError().contains('The issue date must be within 3 months of the inclusion notice submission date');
+      aboutFacilityUnissued.errorSummary().contains('The issue date must be within 3 months of the inclusion notice submission date');
+      aboutFacilityUnissued.issueDateDay().clear();
+      aboutFacilityUnissued.issueDateMonth().clear();
+      aboutFacilityUnissued.issueDateYear().clear();
+      aboutFacilityUnissued.issueDateDay().type(oneMonthDay);
+      aboutFacilityUnissued.issueDateMonth().type(oneMonthMonth);
+      aboutFacilityUnissued.issueDateYear().type(oneMonthYear);
+      aboutFacilityUnissued.shouldCoverStartOnSubmissionNo().click();
+      aboutFacilityUnissued.coverStartDateDay().type(twentyEightDay);
+      aboutFacilityUnissued.coverStartDateMonth().type(twentyEightMonth);
+      aboutFacilityUnissued.coverStartDateYear().type(twentyEightYear);
+      aboutFacilityUnissued.continueButton().click();
+      aboutFacilityUnissued.coverStartDateError().contains('Cover start date cannot be before the issue date');
+      aboutFacilityUnissued.errorSummary().contains('Cover start date cannot be before the issue date');
+      aboutFacilityUnissued.coverStartDateDay().clear();
+      aboutFacilityUnissued.coverStartDateMonth().clear();
+      aboutFacilityUnissued.coverStartDateYear().clear();
+      aboutFacilityUnissued.coverStartDateDay().type(threeMonthsOneDayDay);
+      aboutFacilityUnissued.coverStartDateMonth().type(threeMonthsOneDayMonth);
+      aboutFacilityUnissued.coverStartDateYear().type(threeMonthsOneDayYear);
+      aboutFacilityUnissued.continueButton().click();
```

Comment:
Can we add spaces between each field and each submit button click - or split up into seperate tests for each error scenario? This is very hard to read

Reply:
done